src/benchmarks/micro/runtime/System.Reflection/Invoke.cs

```diff
@@ -83,80 +88,137 @@ public void Method0_NoParms()
         // Include the array allocation and population for a typical scenario.
```

Comment:
did you intentionally not change Method0_NoParms()?

Reply:
Yes. Thanks. Fixed.